lib/acl.c

```diff
@@ -75,6 +77,22 @@ struct rte_acl_field_def ipv6_defs[NUM_FIELDS_IPV6] = {
 		.offset = offsetof(struct ipv6_hdr, dst_addr[12]),
 	},
 	{
+		.type = RTE_ACL_FIELD_TYPE_BITMASK,
+		.size = sizeof(uint16_t),
+		.field_index = SRCP_FIELD_IPV6,
+		.input_index = PORT_INPUT_IPV6,
+		.offset = sizeof(struct ipv6_hdr) +
+			offsetof(struct tcp_hdr, src_port),
```

Comment:
The source and destination ports are the first and second fields in TCP and UDP. You can use this fact to drop the header file `rte_tcp.h`. Moreover, without this property, the name `SRCP_FIELD_IPV6` wouldn't be appropriate since it wouldn't identify the protocol to which it refers.

This property of the ports is not obvious, so please add a comment stating it.